docs/conf.py

```diff
@@ -89,7 +89,7 @@
 
 html_context = {
     'current_language': 'English',
-    'languages': (('English', '/latest'), ('Chinese', '/cn/latest')),
+    'languages': (('English', '/latest'), ('中文', '/cn/latest')),
```

Comment:
```suggestion
    'languages': (('English', '/latest'), ('简体中文', '/cn/latest')),
```
Just to be consistent with conf_zh_CN.py, using "Simplified Chinese" as the language and not changing /cn in the folder structure.  :)